ckanext/canada/tables/ati.yaml

```diff
@@ -4,8 +4,13 @@ target_dataset: ati
 title: ATI Summaries
 notes: Access, upload and modify the monthly ATI Summaries and ATI Nothing to Report for your organization
 
+upload_info:
+  en: "The ATI Summaries template has been updated as of April 2021. You will need to use the new version of the template to upload reports. Please email open-ouvert@tbs-sct.gc.ca with any questions."
+  fr: "Le modèle de sommaire de demande d’accès à l’information a été révisé en avril 2021. Vous devrez utiliser la nouvelle version pour télécharger vos rapports. Si vous avez des questions, veuillez communiquer avec open-ouvert@tbs-sct.gc.ca."
+
+
 template_version: 3
-template_updated: TBD
```

Comment:
I normally add this date right before deploying to production

Reply:
oh, is this a commitment for when we will go live with the changes?

Reply:
Yep - hoping to go live on the 19th with the template changes, it can wait until we actually do it though, I'm not picky